docs/adding_packages/dependencies.md

```diff
@@ -182,10 +182,12 @@ Currently, these are:
 * OpenSSL: `[>=1.1 <4]` for libraries known to be compatible with OpenSSL 1.x and 3.x
 * CMake: `[>3.XX <4]`, where `3.XX` is the minimum version of CMake required by the relevant build scripts
 * Libcurl: `[>=X.YY <9]`, where `X.YY` is the minimum version of Libcurl required, starting from `7.78`
+* Zlib: `[>=1.2.11 <2]`
+* Libpng: `[>=1.X <2]`, where `1.X` is the minimum version of libpng required by the recipe
 
-> **Note**: You might also see Zlib ranges in some PR by CCI maintainers.
+> **Note**: You might also see Libpng & Libxml ranges in some PR by CCI maintainers.
```

Comment:
"You might also see version ranges in some PRs by CCI maintainers. These are being done on a case-by-case basis, and are being rolled out in phases to ensure that they do not cause problems to users."

(i.e. just because it's in a single recipe by us doesn't mean it should/can be in all recipes _unless it's on this list_))